molecule/resources/provisioning/AWS/create.yml

```diff
@@ -2,7 +2,7 @@
 - name: Create
```

Comment:
That file should be removed. We use external molecule drivers from its own repository.